core/tests/webdriverio_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -411,8 +411,50 @@ describe('Core exploration functionality', function() {
       await explorationPlayerPage.expectExplorationToBeOver();
     });
 
+  it('should not show self-loop warning when current and ' +
```

Comment:
I'm a bit torn about this test. It is oddly specific and I feel like a developer reading this in the future will have no idea what we're talking about here.

I think maybe you can instead write something like "it shows no self-loop warnings when navigating between different states" or similar.

Reply:
Made the requested changes. PTAL